File: OLP_ICPC/Python/count_digit.py
def count_digit(x, k):
	if x == k == 0:
		return 1
	ans = 0
	while x:
		ans += (1 if x % 10 == k else 0)
		x //= 10
	return ans

t = int(input())
while t:
	t -= 1
	n, k = map(int, input().split())
	A = list(map(int, input().split()))
	ans = 0
	for a in A:
		ans += count_digit(a, k)
	print(ans)

File: OLP_ICPC/C++/count_digit.cpp
#include <iostream>
using namespace std;

int count_digit(int x, int k) {
	if (x == 0 && k == 0)
		return 1;
	int ans = 0;
	while (x) {
		if (x % 10 == k)
			++ans;		
		x /= 10;
	}
	return ans;
}

int main() {
	int t, n, k;
	cin >> t;
	while (t--) {
		cin >> n >> k;
	}
	cout << count_digit(14, 1);
}